Styles.css

```diff
@@ -282,6 +292,87 @@
   background-color: var(--clothing-red-color);
 }
 
+.testimonials__carousel {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-top: calc(var(--margin-spacer) * 3);
+  font-size: calc(var(--font-size) * 1);
+  display: none;
```

Comment:
why is this here?

Reply:
this is used in order to toggle the testimonial that is shown in the carousel. It starts off with none displayed then I give it another class that changes the display to show it once the button is clicked